zerver/views/pointer.py

```diff
@@ -52,3 +52,12 @@ def get_profile_backend(request, user_profile):
         result['max_message_id'] = messages[0].id
 
     return json_success(result)
+
+def deactivate_user_backend(request, user_profile):
+    # type: (HttpRequest, UserProfile) -> HttpResponse
+    return _deactivate_user_profile_backend(request, user_profile)
+
+def _deactivate_user_profile_backend(request, target):
+    # type: (HttpRequest, UserProfile) -> HttpResponse
+    do_deactivate_user(target)
+    return json_success({})
```

Comment:
Why do you have two functions here?  I think you could just write `deactivate_user_backend` doing everything.

Also, I notice that there are functions with the same names used for the "admins can deactivate other users" feature in the /#administration page in `zerver/views/users.py`; you may want to move this there and trying to reuse that code.

Also, one detail -- we probably want to not allow deactivating the last administrator in an organization -- can you add a check to make sure that isn't happening?